apps/i18n/common/en_us.json

```diff
@@ -1522,6 +1522,8 @@
   "noAssessments": "It looks like there are no multi-question assessments or surveys in this course. Instead, you can measure the students’ progress using the 'Progress' tab. If you are interested in giving your students additional assessments, you can find recommended questions and areas in the lesson plans.",
   "noClassroomsFound": "No classrooms found.",
   "noColumnsInTable": "We couldn't find any columns in \"{table}\". Make sure this table is imported in your project.",
+  "noCurriculumSearchResultsBody": "None of our curricula match your exact criteria, but many of our offerings are flexible! Try broadening your search or consider building a custom curriculum from our more modular options (i.e. teaching two quarter-long curricula for a semester).",
```

Comment:
Late to the game here, and this is another copy edit ... I believe the parens should be "e.g." (for example) instead of "i.e." (in other words). Not sure if this matters at all but it did catch my eye as I've been catching up @dmcavoy

Reply:
Oh good call! I'll ask Dani and put in a fix for that! Thank you!

Reply:
Thank you Meg. I appreciate having people who are strong copy editors looking at this stuff because its definitely not a strength of mine. Thanks for taking the time to review this even after it was merged!